content/en/datadog_cloudcraft/_index.md

```diff
@@ -0,0 +1,158 @@
+
+Enable the **Show All Controls** toggle to display the available **Group By** options. You can also remove specific groupings by unchecking options like VPC and Region. To view the current nesting structure and add the Network ACL (Network Access Control List) layer, click the **More** dropdown.
+
+{{< img src="datadog_cloudcraft/cloudcraft_group_by_2.mp4" alt="Video showing the Group by feature in Cloudcraft" video=true >}}
+
+## Presets
+
+Presets offer a convenient way to apply predefined sets of group-bys and filters, allowing you to view your resources from different perspectives. This feature simplifies the process of applying groupings and filters to your diagrams, allowing you to focus on specific aspects of your architecture.
+
+Cloudcraft provides three built-in presets: [Infrastructure](#infrastructure-diagram), [Network](#network-diagram), and [Security](#security-diagram). These views are designed to address different operational needs.
+
+{{< img src="datadog_cloudcraft/presets.png" alt="Screenshot of the three available presets in Cloudcraft" style="width:80%;" >}}
+
+### Infrastructure diagram
+
+The infrastructure view provides a broad overview, grouping resources by Account, Region and VPC. This view is ideal for generating architecture diagrams for troubleshooting or high-level review.
+
+The infrastructure diagram excludes components like EBS, NAT Gateway, and Transit Gateway, among others, to give you an uncluttered diagram, showing you the most important parts of your architecture.
+
+{{< img src="datadog_cloudcraft/cloudcraft_infra_diagram_2.png" alt="Screenshot of the Infrastructure diagram in Cloudcraft" style="width:100%;" >}}
+
+### Network diagram
+
+The network view adds granularity by introducing Subnet grouping, making it especially useful for network teams seeking to identify latency sources and traffic patterns. In addition, a component may appear multiple times if they belong to multiple subnets.
+
+This diagram excludes components such as EBS, S3, and SNS.
+
+{{< img src="datadog_cloudcraft/cloudcraft_network_diagram.png" alt="Screenshot of the Network diagram in Cloudcraft" style="width:100%;" >}}
+
+### Security diagram
+
+The security view focuses on potential security exposures, grouping resources by Region, VPC, and Security Group. This view is essential for identifying security risks and understanding rules governing inbound and outbound service communications, and is perfect for mapping attack surfaces during penetration testing or security audits. 
+
+This diagram excludes EBS, NAT Gateway, and other components that might clutter the security view. 
+
+**Note**: By default, when you select the Security diagram view, the [Overlay](#overlays) feature defaults to **Security Findings**.
+
+{{< img src="datadog_cloudcraft/cloudcraft_security_diagram.png" alt="Screenshot of the Security diagram in Cloudcraft" style="width:100%;" >}}
+
+### Saved views 
+
+To apply a saved view to your diagram:
```

Comment:
This is the first mention of saved views, just wanted to verify that the customer is expected to already understand what these are?

Reply:
thanks great callout ! I've added more copy here